security/custom_authenticator.rst

```diff
@@ -130,6 +130,12 @@ The authenticator can be enabled using the ``custom_authenticators`` setting:
             ;
         };
 
+.. deprecated:: 5.4
+
+    Having multiple user providers and a ``custom_authenticator`` and not having specified the user provider 
+    to use has been deprecated in Symfony 5.4. If you have registered multiple user providers, you must set the 
+    ``provider`` key to one of the configured providers, even if your custom authenticators don't use it.
```

Comment:
I think I would reword this a bit:

>  If you have registered multiple user providers, you must set the 
    ``provider`` key to one of the configured providers, even if your custom authenticators don't use it. Not doing so is deprecated in Symfony 5.4.

Reply:
I agree, it's done